### PR TITLE
Set default number of entries

### PR DESCRIPTION
**Pull request by wtachau**

### CODE REVIEW

File: graphql-server/migrations/metadata.yaml [hunk 1]
```diff
@@ -0,0 +1,379 @@
+version: 2
```
Comment (wtachau) [inline graphql-server/migrations/metadata.yaml:1]: not sure why this got included -- perhaps i'm on a different version? or should this be gitignored?

Comment (avimoondra) [reply graphql-server/migrations/metadata.yaml:1]: Looks like migration v2 config - part of 1.2.0 release - https://github.com/hasura/graphql-engine/releases/tag/v1.2.0

I'd like to get here asap, probably after this weekend to not disrupt peak activity.

See https://trello.com/c/2SRKKk1A/67-xs-upgrade-hasura-image-to-v120-https-githubcom-hasura-graphql-engine-releases-tag-v120 and https://trello.com/c/MRVL1bPt/68-s-update-migrations-from-v1-v2-after-120.

Weird that it generated for you - Are you using the docker image from docker-compose.yaml? 

IF so, we may need to pin `graphql-server/config.yaml` at 1.0

Comment (avimoondra) [reply graphql-server/migrations/metadata.yaml:1]: For now - I'll remove this file. But we should version control it, when we are 2.0 in the future.

Comment (wtachau) [reply graphql-server/migrations/metadata.yaml:1]: ahh, i see. I did update the hasura-cli when it prompted me to do so -- didn't realize that would have tangible effects. will revert.

Comment (wtachau) [reply graphql-server/migrations/metadata.yaml]: fwiw, i think that's what would happen if someone installed hasura today via
```sh
curl -L https://github.com/hasura/graphql-engine/raw/master/cli/get.sh | bash
```

Comment (avimoondra) [reply graphql-server/migrations/metadata.yaml]: Got it - will see if there is a workaround for that

File: graphql-server/migrations/1588309975119_alter_table_public_games_alter_column_num_entries_per_player/up.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+- args:
+    cascade: false
+    read_only: false
+    sql: ALTER TABLE "public"."games" ALTER COLUMN "num_entries_per_player" TYPE int4;
+  type: run_sql
+- args:
+    cascade: false
+    read_only: false
+    sql: ALTER TABLE ONLY "public"."games" ALTER COLUMN "num_entries_per_player" SET
+      DEFAULT 6;
```
Comment (avimoondra) [inline graphql-server/migrations/1588309975119_alter_table_public_games_alter_column_num_entries_per_player/up.yaml:10]: image

image

Comment (avimoondra) [reply graphql-server/migrations/1588309975119_alter_table_public_games_alter_column_num_entries_per_player/up.yaml:10]: cc: @wtachau I went back and forth b/w 4 and 6... and settled on 6 haha 😆

**Comment by avimoondra** [comment]
[[M-L] Improve Host/Join Experience](https://trello.com/c/NwI9t8lG/6-m-l-improve-host-join-experience)

**Pull request merged**